README.md

```diff
@@ -44,6 +45,65 @@ We plan to remove the need for this work-around via [OpenSearch#581](https://git
 
 To upgrade your existing plugins to work with OpenSearch, see [UPGRADING](./UPGRADING.md).
 
+### Installing Plugins
+
+#### OpenSearch Plugins
+
+Assemble, extract and run OpenSearch `1.0.0-beta1` using [Building OpenSearch package](https://github.com/opensearch-project/OpenSearch/blob/main/TESTING.asciidoc#creating-packages).  
+
+_Example_: For Linux platform.
+
+```
+~/ > git clone https://github.com/opensearch-project/OpenSearch.git
+~/OpenSearch (main)> git checkout 1.0.0-beta1
+~/OpenSearch (main)> ./gradlew :distribution:archives:linux-tar:assemble -Dbuild.version_qualifier=beta1 -Dbuild.snapshot=false
+~/OpenSearch (main)> tar vfxz distribution/archives/linux-tar/build/distributions/opensearch-1.0.0-beta1-linux-x64.tar.gz
+~/OpenSearch (main)> cd opensearch-1.0.0-beta1
+~/OpenSearch/opensearch-1.0.0-beta1 (main)> ./bin/opensearch
+```
+
+Checkout, build and install the plugin.
+
+_Example_: Install Anomaly Detection(Job Scheduler plugin which is a dependency).
```

Comment:
Install Anomaly Detection and Job Scheduler plugins

Reply:
:/ I missed this comment. Will send out another change later.

Reply:
nbd